src/storage/FileResourceStore.ts

```diff
@@ -208,16 +196,16 @@ export class FileResourceStore implements ResourceStore {
           } catch (_) {
             // Metadata file doesn't exist so lets keep `rawMetaData` an empty array.
           }
-          return {
-            dataType: DATA_TYPE_BINARY,
-            data: streamifyArray(quads).pipe(new StreamWriter({ format: 'text/turtle' })),
+
+          resolve({
+            dataType: DATA_TYPE_QUAD,
+            data: streamifyArray(quads),
             metadata: {
               raw: rawMetadata,
               profiles: [],
-              contentType: 'text/turtle',
```

Comment:
There actually is a contentType for having a stream of quads, it's `internal/quads` (and it has a corresponding constant in `util/ContentTypes.ts`). Some other classes depend on this so it's important to always have a content type.